Files/App.xaml

```diff
@@ -104,11 +115,11 @@
                             <x:Double x:Key="SidebarTintOpacity">0.8</x:Double>
                             <x:Double x:Key="SidebarTintLuminosityOpacity">0.8</x:Double>
                             <!--  Tabs  -->
-                            <SolidColorBrush x:Key="TabViewItemHeaderBackground" Color="{StaticResource SystemControlBackgroundBaseLow}" />
-                            <SolidColorBrush x:Key="TabViewItemHeaderBackgroundSelected" Color="{StaticResource SystemControlHighlightChromeHigh}" />
-                            <SolidColorBrush x:Key="TabViewItemHeaderBackgroundPressed" Color="{StaticResource SystemControlHighlightChromeHigh}" />
-                            <SolidColorBrush x:Key="TabViewItemHeaderBackgroundPointerOver" Color="{StaticResource SystemControlHighlightChromeHigh}" />
-                            <SolidColorBrush x:Key="TabViewItemSeparator" Color="{StaticResource SystemControlForegroundBaseLow}" />
```

Comment:
Is there a visual change?

Reply:
@yaichenbaum Those resources weren't available in HighContrast and would crash the app for some reason. It's best to use the correct resources detailed as per the accessibility documentation for Windows apps.